sdk/couchbase-core/src/memdx/datatype.rs

```diff
@@ -6,6 +6,12 @@ pub enum DataTypeFlag {
     Xattrs,
 }
 
+impl Default for DataTypeFlag {
```

Comment:
NIT: Not a blocker, but we could use `#[derive(Default)]` for this

Reply:
I don't think that'd work here would it? How would it know which variant is the default value?